examples/disaggregated/slurm/cache_transceiver_test/run_cache_transceiver_test.py

```
@@ -366,60 +540,108 @@ def run_one_request(
-        if state == LlmRequestState.DISAGG_TRANS_ERROR:
-            raise _TransferError("ctx transfer reported DISAGG_TRANS_ERROR")
         return None
 
     # gen side
+    initial_error = None
     if is_leader:
-        zmq_sock.send(b"go")
-        status, payload = pickle.loads(zmq_sock.recv())
+        try:
+            zmq_sock.send(b"go")
+            status, payload = pickle.loads(zmq_sock.recv())
```

Ruff (0.16.1)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)